### PR TITLE
fix memcost example

### PR DESCRIPTION
**Pull request by yajiedesign**
recovery NNVM_EXEC_ENABLE_INPLACE env check
fixed https://github.com/apache/incubator-mxnet/issues/6272

### CODE REVIEW

File: src/executor/inplace_addto_detect_pass.cc [hunk 1]
```diff
@@ -32,6 +32,7 @@ namespace mxnet {
 namespace exec {
 
 Graph DetectInplaceAddTo(Graph g) {
+  size_t enable_inplace = dmlc::GetEnv("NNVM_EXEC_ENABLE_INPLACE", true);
```
Comment (eric-haibin-lin) [inline src/executor/inplace_addto_detect_pass.cc]: Why not use `MXNET_EXEC_ENABLE_INPLACE ` since the pass is in MXNet?

Comment (yajiedesign) [reply src/executor/inplace_addto_detect_pass.cc]: @piiswrong

**Comment by piiswrong** [comment]
I think inplace mostly happens in planmemory. not here

**Comment by eric-haibin-lin** [comment]
@yajiedesign could you also update the plan_memory pass in nnvm?
The option was there before v0.9 nnvm refactoring. https://github.com/apache/incubator-mxnet/blob/cf84037c7a55f3bc98fec447ae3a39929030f002/src/symbol/graph_executor.cc#L406 

If this option is available, issues like "kAddTo is not implemented for operator X" can use this as a walk-around, which is great.

**Comment by piiswrong** [comment]
@yajiedesign Could you update?

**Comment by piiswrong** [comment]
closing due to inactive.
@eric-haibin-lin Please take a look while you are investigating memory cost issue

**Comment by eric-haibin-lin** [comment]
ok

**Pull request closed without merging**